src/plugins/formvalid/formvalid-fr.hbs

```diff
@@ -425,7 +425,7 @@
 		&lt;/fieldset&gt;</code></pre>
 			</details>
 
-			<input type="submit" value="Soumettre" class="btn btn-primary" /> <input type="reset" value="Réinitialiser" class="btn btn-default" />
+			<input type="submit" value="Soumettre" class="btn btn-primary" /> <input type="reset" value="Réinitialiser la page aux valeurs par défaut" class="btn btn-link pull-right" />
```

Comment:
Not sure regarding to pull the reset button to the right. That would fail the straw test.

Reply:
Made have it under the submit button, but with a decent amount of spacing between them to prevent accidental clicking?

Reply:
I think rendering the reset button as a link would be sufficient.

@GormFrank @bsouster What do you think?

Reply:
Any updates? Do you agree with my opinion @duboisp and @thekodester?